cmd/cloud_metrics.go

```diff
@@ -0,0 +1,134 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"slices"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"go.k6.io/k6/cloudapi"
+)
+
+func getCloudMetricsCmd(client *cloudapi.K6CloudClient) *cobra.Command {
+	metricsSub := &cobra.Command{
+		Use:   "metrics",
+		Short: "Query test run metrics",
+	}
+
+	var testRunID string
+	var ofJson bool
+	listMetrics := &cobra.Command{
+		Use:   "list",
+		Short: "List metrics in the test run",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			metrics, err := client.GetCloudTestRunMetrics(testRunID)
+			if err != nil {
+				return err
+			}
+			slices.SortFunc(metrics, func(a, b cloudapi.Metric) int {
+				switch {
+				case a.Origin == b.Origin:
+					return strings.Compare(a.Name, b.Name)
+				case a.Origin == "builtin":
+					return -1
+				case b.Origin == "builtin":
+					return 1
+				default:
+					return strings.Compare(a.Origin, b.Origin)
+				}
+			})
+			out := NewCloudOutput("", []string{"Name", "Type", "Origin"})
+			for _, metric := range metrics {
+				out.Add(map[string]any{
+					"Name":   metric.Name,
+					"Type":   metric.Type,
+					"Origin": metric.Origin,
+				})
+			}
+
+			if ofJson {
```

Comment:
nitpick: I wonder if we should call it outputJson or similar to be more readable 🤔 
I did in my PR but we can keep it as it is for consistency

Reply:
Thanks! Let's address this separately